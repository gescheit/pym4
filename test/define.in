define(foo,bar)
foo foo
foofoo
foo`foo'foo
define(bar,xyz)
foo foo
foofoo
foo`foo'foo
