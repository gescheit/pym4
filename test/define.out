
bar bar
foofoo
barfoobar

xyz xyz
foofoo
xyzfooxyz
